docs/api-docs/building-checkouts/checkout-consignment.md

```diff
@@ -39,9 +39,9 @@ For more information on OAuth Scopes and authentication, see [Authentication](ht
 
 > For limits on the number of line items in a consignment, see the [Create a consignment endpoint](/api-reference/storefront/checkouts/checkout-consignments/checkoutsconsignmentsbycheckoutidpost) documentation.
 
-> The Storefront Checkout API allows developers to build a checkout experience by accessing raw API request/response data.
-
-> The Server-to-Server Checkout API responds quickly when the checkout contains one consignment.  Each additional consignment increases the amount of time the API takes to finish creating a checkout.
+> The Server-to-Server Checkout API responds quickly when the checkout contains one consignment.  Each additional consignment increases the amount of time the API takes to finish creating a checkout. The example API calls in this article use the Server-to-Server Checkout API.
+  
+> The Storefront Checkout API allows developers to manage a shopper’s cart and checkout and fetch order data via client-side JavaScript on BigCommerce Stencil powered storefronts. OAuth scopes are not needed.
```

Comment:
This sentence does a lot of work and could be more digestible.  

* The style guide prefers `using` to `via`.  
* There's a list of three not-quite parallel things here -- developers manage (a shopper's) cart, checkout, and fetch order data.  
* Perhaps the `client-side JavasScript ... Stencil` clause should move up to the front because it seems like the Storefront Checkout API is not going to apply to headless checkout flows.  
* There should be a way to make OAuth scopes an object somewhere, rather than a subject, so they don't have to be `not needed` in the passive voice.